webview-ui/src/ClusterProperties/ClusterDisplay.tsx

```diff
@@ -14,6 +14,8 @@ export interface ClusterDisplayProps {
 
 type StartStopState = "Started" | "Starting" | "Stopped" | "Stopping";
 
+const terminalProvisioningStates = ["Canceled", "Failed", "Succeeded", "Starting", "Stopping"];
```

Comment:
I think this means we will never be able to abort start/stop operations - is that right? Are we sure about that restriction?

If that's what we want, I'd prefer to make it explicit in the code and keep `terminalProvisioningStates` consistent with the agent pool level ones (i.e. just `["Canceled", "Failed", "Succeeded"]`).

If we want to prevent users aborting start/stop operations, could we have a separate variable for this, something like:
```typescript
const unabortableProvisioningStates = [...terminalProvisioningStates, "Starting", "Stopping"];
```

And then later:
```typescript
const showAbortButton = !unabortableProvisioningStates.includes(props.clusterInfo.provisioningState);
```